simulators/kcc/test-flag/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"context"
+	"math/big"
+	"strconv"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/hive/hivesim"
+)
+
+var (
+	TestnetGenesisHash = common.HexToHash("0x3e98a8f1704be5accc0ce9abf3e852e24ec030a9dda6c1beaf72c600f85cd417")
```

Comment:
The hash of the tesenet genesis is taken from https://github.com/kcc-community/kcc/blob/v1.2.0/params/config.go#L32